documentation/docs/components/video.md

```diff
@@ -0,0 +1,18 @@
+# Video
+
+## Example
+
+| type     | name       |parameter_list |
+|--------- | -----------| ------        |
+|video     |video_src	|subtitles_src: plh_video/lets_slow_down.vtt |
+
+![](images/video.png)
+
+[Google Sheet Demo](https://docs.google.com/spreadsheets/d/1OjY24TnAWRxiiPT9vED59DLwz2Zb9RmUHw-Lym1pCXo/edit#gid=569531329)   
+[Live Preview Demo](https://plh-global.web.app/template/comp_video)
+
+## Parameters
+
+| Parameter             | Value                        | Description |
+| ---------             | -----------                  | --------- |
+|subtitles_src          |plh_video/lets_slow_down.mp4  | On clicking the play button, the "Let's slow down" song plays|
```

Comment:
`.mp4` is a video format, `.vtt` is a subtitle format. The `subtitles_src` parameter takes a subtitle file as an input. The value column takes a `.mp4` file.